apps/condo/domains/user/constants/errors.js

```diff
@@ -8,16 +7,16 @@ const AUTH_BY_PASSWORD_FAILED_ERROR = '[passwordAuth:failure'
 const MIN_PASSWORD_LENGTH_ERROR = '[register:password:minLength'
 const RESET_TOKEN_NOT_FOUND = '[resetPassword:token:notFound'
 const PASSWORD_TOO_SHORT = '[password:min:length'
-
```

Comment:
these spaces are used to logically group the code

Reply:
Done